config/features/config.go

```diff
@@ -154,6 +161,10 @@ func applyPraterFeatureFlags(ctx *cli.Context) {
 func applySepoliaFeatureFlags(ctx *cli.Context) {
 }
 
+// Insert feature flags within the function to be enabled for Holesky testnet.
+func applyHoleskyFeatureFlags(ctx *cli.Context) {
```

Comment:
i think deepsource is complaining these parameters are not used

Reply:
I think it's fine. These are essentially reserved functions for future use anyway